src/test/scala/projectEuler/LargestPrimeFactorCalculatorSpec.scala

```diff
@@ -0,0 +1,42 @@
+package projectEuler
+
+import org.scalatest.{Matchers, WordSpec}
+
+/**
+ * Created by sonia on 12/10/2015.
+ */
+class LargestPrimeFactorCalculatorSpec extends WordSpec with Matchers {
```

Comment:
Nice scalatest wordspec usage :+1: If you wanted to be clever, you could parameterise the tests with the number and expected factors so that you didn't need to repeat the tests